src/commonMain/kotlin/fr/acinq/lightning/wire/RecipientData.kt

```diff
@@ -0,0 +1,85 @@
+package fr.acinq.lightning.wire
+
+import fr.acinq.bitcoin.ByteVector
+import fr.acinq.bitcoin.PublicKey
+import fr.acinq.bitcoin.io.ByteArrayInput
+import fr.acinq.bitcoin.io.ByteArrayOutput
+import fr.acinq.bitcoin.io.Input
+import fr.acinq.bitcoin.io.Output
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class RecipientDataTlv : Tlv {
```

Comment:
Can you put this in a file called `RouteBlinding.kt` inside `wire` and rename it `RouteBlindingEncryptedDataTlv` to match what is in `eclair`? It's best to be as close as possible to `eclair` to make it easy to spot differences and reduce maintenance burden.

Reply:
Done